### ISSUE TITLE
Leaflet map coordinate issue

### ISSUE THREAD

**Issue by webgisdeveloper** [GeoGateway/geogateway-django-app#114]
From Jay's email: 9/27/2021  

> Another thought on the -180to 180 cycle problem in mapping:  
I simple and helpful interim solution could be to put a button in a corner of the map labeled “whole earth”, and the response to a click would be to display the map from lon -180 to 180, lat -80 to  80 or -90 to 90. This would support users who want to plot global information, and also as a handy intermediate step before zooming into to a particular location.   

> A related development would add a related selection option (check box?) to the various mapping tabs (GNSS, UAVSAR, Seismicity, etc).  This could be a “whole earth” selection box adjacent to the “Draw an area on map.”  This would fill the lon, lat span entries with -180 to 180, -90 to 90.   

> These proposed modifications would be particularly helpful for educational applications: it is easy and interesting to display earthquakes > M6 over any 20 year period, to highlight the ring of fire and the mid-atlantic ridge.  It is also interesting to plot the global set of GNSS (JPL) velocities, which together with the >M6 seismicity tells much of the story of plate tectonics for students.
Attempting this with the current system requires considerable risk (the view of the map needs to be centered at 0, 0, which is not simple to select), and risks embarrassing results when views are selected that span the -180 180 longitude cycle jump: only part of the display data is displayed on the map in those cases, similar to the troubles documented in the earlier message (below).
In summary, adding a whole-earth standard map view together with a whole-earth option for data selection, GeoGateway will be more useful for educators and students, and easier to demonstrate live with  much-reduced risk of missing data. A complete solution to the problems for selections boxes crossing the longitude 180 line would also be helpful, but could be implemented later with more care.

**Comment by webgisdeveloper** [comment]
https://leafletjs.com/reference-1.7.1.html#map-option  maxBounds  worldCopyJump  wrapLatLng
